tests/helpers/profile.ts

```diff
@@ -7,12 +7,19 @@ export const randomProfileName = (): string => {
 };
 
 export const createProfile = async (page: Page, profile: string) => {
+  await visitProfileCreation(page, profile);
+  await finishProfileCreation(page, profile);
+};
+
+export const visitProfileCreation = async (page: Page, profile: string) => {
```

Comment:
The name is a bit confusing, because it's not only visiting the page, it's actually *starting* the creation. So maybe something like `startProfileCreation`?